docs/source/usage.rst

```diff
@@ -1,18 +1,18 @@
 .. highlight:: none
 
-Using as a command line tool
-============================
+Using the command line tool
+===========================
 
-The command-line syntax to run the ``nbconvert`` script is::
+``nbconvert`` can be run from the command line using::
 
   $ jupyter nbconvert --to FORMAT notebook.ipynb
 
-This will convert the Jupyter notebook file ``notebook.ipynb`` into the output
+For example, nbconvert will convert the Jupyter notebook file ``notebook.ipynb`` into the output
 format given by the ``FORMAT`` string.
 
 Default output format - HTML
 ----------------------------
-The default output format is HTML, for which the ``--to`` argument may be
+nbconvert's default output format is HTML, and the ``--to`` argument may be
```

Comment:
Ahh this changed actually -- if you do not specify a `--to` you will get `ValueError: Please specify an output format with '--to <format>'.` I didn't realize we missed the doc update for that :/

Reply:
That probably means we need to find all the `jupyter nbconvert ` and change them to `jupyter nbconvert --to=html` unless they have a `to` argument. I can take that after this merges if you like.